### PR TITLE
L1T online and offline DQM configuration upgrades

### PR DESCRIPTION
**Pull request by thomreis**
L1T online and offline DQM improvements.

- Move the TF emulators and the uGT emulator before the fat event filter to have better statistics for the comparisons. This means the modules are then also ran in the offline DQM.
- Ignore BX range mismatches for BMTF and EMTF data to emulator comparison in the summary.
- Remove the old unpacker for CaloLayer1 and use the rawToDigi standard sequence instead.
- Remove unneeded unpackers from the online DQM clients.
- Add the ESProducer for the ideal magnetic field to the online client because it is needed by the EMTF emulator.

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by thomreis** [comment]
Hi @jfernan2 , @dmitrijus 
could you take a look at this PR and its backport please and let me know if things are OK?

**Comment by dmitrijus** [comment]
This adds a half a megabyte worth of histograms, is this intentional?

**Comment by thomreis** [comment]
Yes, this should add additional histograms to the L1TEMU directory in the offline DQM. Previously the modules have not been running since they were in a sequence that was only after the fat event filter in the online DQM.

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
hi @dmitrijus - to clarify its 0.5MB spread across 23 files.. (probably this monitoring needs to also track the biggest single increase)

File: DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -34,6 +34,21 @@
 
 process.load("Configuration.StandardSequences.RawToDigi_Data_cff")    
 
+# remove unneeded unpackers
+process.RawToDigi.remove(process.ecalPreshowerDigis)
```
Comment (davidlange6) [inline DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py]: its probably more maintainable to include the unpackers you want instead...

**Comment by davidlange6** [comment]
+1

**Pull request merged**